common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -151,11 +153,18 @@
                 });
             };
 
-            DiscussionThreadListView.prototype.addSearchAlert = function(message) {
+            /**
+             * Creates search alert model and adds it to collection
+             * @param message - alert message
+             * @param css_class - css class for a message
```

Comment:
Nit: "css class for a message" isn't a very helpful explanation :-p . That says what but not why.